src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL_COUNT = 100;
```

Comment:
name is too difficult - no need to use both words `number` and `count`

let's simplify it to `MAX_BALL_NUMBER`